src/sql/workbench/services/notebook/common/localContentManager.ts

```diff
@@ -25,8 +24,12 @@ export class LocalContentManager implements nb.ContentManager {
 	constructor(@IFileService private readonly fileService: IFileService) { }
 
 	public async loadFromContentString(contentString: string): Promise<nb.INotebookContents> {
-		let contents: JSONObject = json.parse(contentString);
```

Comment:
Do you remember the reason we chose to use the vscode json parser? what are the possible negative impacts this change might introduce?

Reply:
To be honest, I’m not sure that any of us had a good grasp on using the various parsers when this was written.

It’s possible that malformed JSON will now throw an error. I’m not aware of any specifics beyond that, but obviously this is something that we should test the heck out of.

Reply:
(by the way the failing test is… invalid json 😛)